src/lib/apollo/link.ts

```diff
@@ -51,11 +51,13 @@ export const batchLink = split(
   restProxyHttpLink,
 );
 
-export const makeAuthLink = (apiToken: string) =>
+export const makeAuthLink = (apiToken: string, language?: string | null) =>
   new ApolloLink((operation, forward) => {
+    const languageHeader = language ? { 'Accept-Language': language } : {};
```

Comment:
Doesn't the server automatically honor the user's language preference? This code looks like it uses the language preference and defaults to the accept header: https://github.com/CruGlobal/mpdx_api/blob/7eff744504fe52358a0a975cb5f36ee6b5f1e5ad/app/controllers/graphql_controller.rb#L59-L68

Reply:
If I'm reading it right, `scope_request_to_locale` runs for every GraphQL request https://github.com/CruGlobal/mpdx_api/blob/7eff744504fe52358a0a975cb5f36ee6b5f1e5ad/app/controllers/graphql_controller.rb#L2

Reply:
Sorry, I misread it. The server uses the `Accept-Language` header first and falls back to the user's preference. I feel like we should ask the server people to switch that. In the meantime, can you just set `Accept-Language` to an empty string? When I do that, translated constants are coming through. I think that would be a simpler solution.